packages/babel-types/README.md

```diff
@@ -1875,6 +1875,22 @@ Aliases: `TSTypeElement`
 
 ---
 
+### tSConditionalType
```

Comment:
<code>t**s**ConditionalType</code>?

Reply:
It's generated, and should get fixed after we land: https://github.com/babel/babel/pull/6993

Reply:
I'm cool with landing it, we can first make a change to https://github.com/babel/babel-upgrade for it pretty easily